src/flow.c

```diff
@@ -416,7 +416,8 @@ void FlowHandlePacketUpdate(Flow *f, Packet *p, ThreadVars *tv, DecodeThreadVars
 #ifdef CAPTURE_OFFLOAD
     } else {
         /* still seeing packet, we downgrade to local bypass */
-        if (SCTIME_SECS(p->ts) - SCTIME_SECS(f->lastts) > FLOW_BYPASSED_TIMEOUT / 2) {
+        const uint32_t timeout_policy = FlowGetTimeoutPolicy(f);
```

Comment:
This looks more like the duration to process the remaining packets already in the pipe versus the duration to set the bypass offload... Not related to the flow state... right ?